packages/client/src/api/fetcher.ts

```diff
@@ -116,16 +117,17 @@ export async function fetch<
   }
 
   const requestId = response.headers?.get('x-request-id') ?? undefined;
+  const payload = await parsePayload(response);
+  if (response.ok) return payload;
```

Comment:
Are there any side effects here for returning a string potentially?

Reply:
Good question, our endpoints should always return JSON... Will fix it and throw if it's a string.